mono/metadata/debug-mono-ppdb.c

```diff
@@ -172,8 +172,7 @@ mono_ppdb_load_file (MonoImage *image, const guint8 *raw_contents, int size)
 		return NULL;
 	}
 
-// Temporarily disabled to unblock Roslyn
-#if HOST_WIN32 //|| HAVE_SYS_ZLIB
+#if HOST_WIN32 || HAVE_SYS_ZLIB
```

Comment:
On master this is `HOST_WIN32 || HOST_WASM || HAVE_SYS_ZLIB`. It's ok here without wasm because we're not building wasm from this branch, is that correct?

Reply:
Yes, exactly.